### ISSUE TITLE
Incorrect Q-Value calculation in "qrdqn" agent

### ISSUE THREAD

**Issue by alirezakazemipour** [ku2482/fqf-iqn-qrdqn.pytorch#18]
Hi,

First, thank you for providing such a clear and easy-to-follow implementation for some important Distributional RL algorithms.
I found the following line not to be correct according to the QRDQN paper:

https://github.com/ku2482/fqf-iqn-qrdqn.pytorch/blob/11d70bb428e449fe5384654c05e4ab2c3bbdd4cd/fqf_iqn_qrdqn/model/qrdqn.py#L74

I believe it should have been:
```python
q = 1 / 200 * torch.sum(quantiles, dim=1)
```
which corresponds to the following equation in the paper:

![Screenshot from 2022-03-02 14-25-47](https://user-images.githubusercontent.com/32295763/156348684-f6c3848d-2726-459e-bb77-9f4ed5513415.png)

And, here *q<sub>j</sub>* is fixed to 1 / 200.

**Comment by toshikwa** [comment]
`quantiles.mean(dim=1)` is equal to `1 / 200 * torch.sum(quantiles, dim=1)`.

**Comment by alirezakazemipour** [comment]
Yeah, now I see.
Since the probabilities of all quantiles are the same, you're right.
Sorry for the inconvenience.
